### ISSUE TITLE
amuse-kepler depends on ph4, it shouldn't.

### ISSUE THREAD

**Issue by rieder** [amusecode/amuse#443]
```
****************************************************************************************************
Building libraries and codes
****************************************************************************************************
****************************************************************************************************
Building code: kepler, target: all, in directory: build/temp.macosx-10.14-x86_64-2.7/codes/kepler
****************************************************************************************************
make: Map '/Users/rieder/Code/Amuse/packages/amuse-kepler/build/temp.macosx-10.14-x86_64-2.7/codes/kepler' wordt binnengegaan
/Users/rieder/Env/Amuse2pip/share/amuse/build.py --type=c interface.py KeplerInterface -o worker_code.cc
/Users/rieder/Env/Amuse2pip/share/amuse/build.py --type=h interface.py KeplerInterface -o interface.h
make -C src all CXX='mpicxx'
make[1]: Map '/Users/rieder/Code/Amuse/packages/amuse-kepler/build/temp.macosx-10.14-x86_64-2.7/codes/kepler/src' wordt binnengegaan
make[1]: *** Er is geen regel om doel '../../ph4/src/kepler.cc' te maken, nodig voor 'kepler-lib.o'.  Gestopt.
make[1]: Map '/Users/rieder/Code/Amuse/packages/amuse-kepler/build/temp.macosx-10.14-x86_64-2.7/codes/kepler/src' wordt verlaten
make: *** [Makefile:31: src/libkepler.a] Fout 2
make: Map '/Users/rieder/Code/Amuse/packages/amuse-kepler/build/temp.macosx-10.14-x86_64-2.7/codes/kepler' wordt verlaten
****************************************************************************************************
********************************************************************************
Building finished
********************************************************************************
```

**Comment by rieder** [comment]
Same goes for smalln:
```
****************************************************************************************************
Building libraries and codes
****************************************************************************************************
****************************************************************************************************
Building code: smalln, target: all, in directory: build/temp.macosx-10.14-x86_64-2.7/codes/smalln
****************************************************************************************************
make: Map '/Users/rieder/Code/Amuse/packages/amuse-smalln/build/temp.macosx-10.14-x86_64-2.7/codes/smalln' wordt binnengegaan
/Users/rieder/Env/Amuse2pip/share/amuse/build.py --type=c interface.py SmallNInterface -o worker_code.cc
/Users/rieder/Env/Amuse2pip/share/amuse/build.py --type=h interface.py -i amuse.support.codes.stopping_conditions.StoppingConditionInterface  SmallNInterface -o interface.h
make -C src all CXX='mpicxx'
make[1]: Map '/Users/rieder/Code/Amuse/packages/amuse-smalln/build/temp.macosx-10.14-x86_64-2.7/codes/smalln/src' wordt binnengegaan
make[1]: *** Er is geen regel om doel '../../ph4/src/smallN.cc' te maken, nodig voor 'smallN-lib.o'.  Gestopt.
make[1]: Map '/Users/rieder/Code/Amuse/packages/amuse-smalln/build/temp.macosx-10.14-x86_64-2.7/codes/smalln/src' wordt verlaten
make: *** [Makefile:33: src/libsmallN.a] Fout 2
make: Map '/Users/rieder/Code/Amuse/packages/amuse-smalln/build/temp.macosx-10.14-x86_64-2.7/codes/smalln' wordt verlaten
****************************************************************************************************
********************************************************************************
Building finished
********************************************************************************
```

**Comment by rieder** [comment]
temporary fix could be to add amuse-ph4 to requirements for these packages?

**Comment by rieder** [comment]
(probably not)

**Comment by ipelupessy** [comment]
fixed

**Comment by ipelupessy** [comment]
I removed the dependency...it was not necessary (everything was already copied...)

**Comment by rieder** [comment]
smalln has the same issue - that's not fixed yet

**Comment by rieder** [comment]
(made that into a separate issue)
